sdk/storage/storage-queue/swagger/README.md

```diff
@@ -12,70 +12,73 @@ enable-xml: true
 generate-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../src/generated
-input-file: ./queue-storage-2019-02-02.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/storage-dataplane-preview/specification/storage/data-plane/Microsoft.QueueStorage/preview/2018-03-28/queue.json
```

Comment:
Only the input file is updated in this file.
Rest all are changes due to formatting.

Reply:
Added some transforms now.